src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -146,6 +147,14 @@ void ResetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
       GetInfo(message, tts_response_info_);
       break;
     }
+    case hmi_apis::FunctionID::RC_SetGlobalProperties: {
+      SDL_LOG_INFO("Received RC_SetGlobalProperties event");
```

Comment:
This event is not triggered in the case of multiple properties being reset. For example
1. Sending SetGlobalProperties request with `userLocation`, `helpPrompt` and `menuTitle`
2. Sending ResetGlobalProperties request with `properties` {USER_LOCATION, HELPPROMPT, MENUNAME}

Reply:
@ShobhitAd we were not able to reproduce this test case. Attached is the ATF script which we used: [3668_App_resets_UserLocation_HelpPrompt_MenuTitle.zip](https://github.com/smartdevicelink/sdl_core/files/6889260/3668_App_resets_UserLocation_HelpPrompt_MenuTitle.zip)

And here is the logs: [2021-07-27_09-58-59.097_3668.tar.gz](https://github.com/smartdevicelink/sdl_core/files/6889264/2021-07-27_09-58-59.097_3668.tar.gz)

As you can see, SDL is properly triggering RC events and handles them in `on_event()`.

Could you please provide your steps on how to reproduce the issue you are currently observing?

Reply:
@AKalinich-Luxoft my reproduction steps are similar to those of the attached ATF test. However, the problem with this test case is that since the **UI** and **TTS** reset responses are successful the overall response (based on how the results are combined) to mobile will also be successful, regardless of the result of **RC** response.

Looking at the attached SmartDeviceLinkCore log file, when I searched for `commands::ResetGlobalPropertiesRequest::on_event: Received` I was only seeing two results - one for the "UI_SetGlobalProperties event" and the other for "TTS_SetGlobalProperties event".

I tried modifying the ATF test to check the case where **UI** and **TTS** return `UNSUPPORTED_RESOURCE` responses and **RC** returns a SUCCESS response and the test fails in this case. Here are the logs for the modified test
[UNSUPPORTED_RESOURCE_test.zip](https://github.com/smartdevicelink/sdl_core/files/6896609/UNSUPPORTED_RESOURCE_test.zip)

Reply:
@ShobhitAd Thank you for your explanation. Good catch. We've extended test coverage in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2534/commits/abb2e314ce21df8b2cbb00005360f1d54d4481f6 Script `3` is failing now by the reason you described.

Reply:
@ShobhitAd this issue is fixed in https://github.com/smartdevicelink/sdl_core/pull/3741/commits/d70d992c7de3752991d99f29451252cda4c4e45f
New ATF script is passed with the new changes done